ext/iperf3/iperf_server_api.c

```diff
@@ -135,6 +138,14 @@ iperf_accept(struct iperf_test *test)
         i_errno = IEACCEPT;
         return -1;
     }
+#ifdef SO_RCVTIMEO
+    /* 30 sec timeout for control socket for param exchange. */
+    tv.tv_sec = 30;
+    tv.tv_usec = 0;
+    if (setsockopt(s, SOL_SOCKET, SO_RCVTIMEO, (struct timeval *)&tv, sizeof(struct timeval)) < 0) {
+		iperf_printf(test, "Unable to set socket SO_RCVTIMEO");
+	}
```

Comment:
**nit** Indentation is a bit off.

Reply:
I actually meant the indentation of the closing curly bracket. :)

Reply:
hmmh, it isn't in my editor. is this some github feature? but btw, code style checks have been disabled from iperf/curl because original is having issues.